--- /usr/share/jolla-gallery/pages/GalleryStartPage.qml
+++ GalleryStartPage.qml
@@ -244,6 +244,17 @@
                 count: model ? model.count : 0
                 type: MediaSource.Photos
             }
+		MediaSource {
+                id: cargoSource
+                title: "Cargo!" + " " + qsTrId("gallery-bt-photos")
+                icon: "PhotoIcon.qml"
+                page: "GalleryGridPage.qml"
+                model: cargoImgModel
+                ready: true
+                busy: model.status == DocumentGalleryModel.Active
+                count: model ? model.count : 0
+                type: MediaSource.Photos
+            }
         }
 
         VerticalScrollDecorator {}
