--- /usr/share/jolla-gallery/gallery.qml
+++ gallery.qml
@@ -61,6 +61,15 @@
         filter: GalleryStartsWithFilter { property: "filePath"; value: androidStorage.path }
     }
 
+    DocumentGalleryModel {
+        id: cargoImgModel
+
+        rootType: DocumentGallery.Image
+        properties: ["url", "mimeType", "title", "orientation", "dateTaken", "width", "height" ]
+        sortProperties: ["-lastModified"]
+        autoUpdate: true
+        filter: GalleryStartsWithFilter { property: "filePath"; value: StandardPaths.download + "/Cargo/" }
+    }
     // For some reason if the gallery is launched via invoker, it sets codecForLocale to
     // be ISO8859-1 instead of UTF-8 which causes the loading failure of the images using
     // Chinese characters. This can be used as a workaround for a bug: JB#11179.
